tasks/e2e-kitchensink.sh

```diff
@@ -110,8 +110,11 @@ create_react_app --scripts-version=$scripts_path --internal-testing-template=$ro
 # Enter the app directory
 cd test-kitchensink
 
+# Link to our preset
+npm link $root_path/packages/babel-preset-react-app
+
 # Test the build
-NODE_PATH=src REACT_APP_SHELL_ENV_MESSAGE=fromtheshell npm run build
+NODE_PATH=src NODE_ENV=test REACT_APP_SHELL_ENV_MESSAGE=fromtheshell npm run build
```

Comment:
why the "test" environment?
This will be used to test the "production" env later on.